client/src/includes/sidePanel.js

```diff
@@ -155,7 +155,10 @@ export default function initSidePanel() {
       newWidth,
     ).replace('%(num)s', newWidth);
 
-    sidePanelWrapper.style.width = `${newWidth}px`;
+    sidePanelWrapper.parentElement.style.setProperty(
+      '--side-panel-width',
+      `${newWidth}px`,
+    );
```

Comment:
Note to self: `--preview-panel-width` could probably be removed in favour of this new variable.